client/app/scripts/liveblog-edit/styles/draft-posts.less

```diff
@@ -40,5 +41,133 @@
             padding: 20px;
             color: @gray;
         }
+        .post-item-holder {
+            padding: 14px 10px 0 12px;
+            border-bottom: 1px solid #DBDBDB;
+            position: relative;
+            .post-item-figure {
+                margin-right: 0;
+                .avatar {
+                    width: 24px;
+                    height: 24px;
+                }
+            }
+            .post-item-content {
+                margin-left: 0px;
+                .post-item__header {
+                    font-size: 12px;
+                    .name {
+                        font-weight: 600;
+                    }
+                }
+                #timeline-dropdown.dropdown-menu {
```

Comment:
@darconny Are you sure that all of that is necessary ?
We should at least rename all classes containing _timeline_ in the name. Because we are in `draft-posts.less`.
Also here we saw an id `#timeline-dropdown`. It shouldn't be an id but a class since it appears several time in the document.